public/api/conf/1.0/application.raml

```diff
@@ -32,19 +32,168 @@ documentation:
           example: "19GBL4592NCOI21NR9"
       /status:
         get:
-          displayName: Get the status of a declaration
-          description: Get the status of a declaration with the given MRN. This endpoint is only available to CSP users.
+          displayName: Get the status of a declaration by MRN
+          description: Get the status of a declaration with the given MRN.
           headers:
             Accept:
               description: Specifies the version of the API that you want to call. See [versioning](/api-documentation/docs/reference-guide#versioning).
               type: string
               required: true
               example: application/vnd.hmrc.1.0+xml
             X-Badge-Identifier:
-                description: Represents the end-user who is interacting with the subscribed application. Must be numeric or upper case letters with a minimum length of 6 and a maximum length of 12.
-                type: string
-                required: false
-                example: ABC123
+              description: Either this or X-Submitter-Identifier (or both) required for CSPs. Optional for non-CSPs. Represents the end-user who is interacting with the subscribed application. Must be numeric or upper case letters with a minimum length of 6 and a maximum length of 12.
+              type: string
+              required: false
+              example: ABC123
+            X-Submitter-Identifier:
+              description: Either this or X-Badge-Identifier (or both) required for CSPs. Optional for non-CSPs. This is the EORI associated with the originating owner of the declaration. Must be numeric and upper case characters with a maximum length of 17 characters.
+              type: string
+              required: false
+              example: GB123456789000
+          (annotations.scope): "write:customs-declarations-information"
+          securedBy: [ sec.oauth_2_0: { scopes: [ "write:customs-declarations-information" ] } ]
+          responses:
+            200:
+              body:
+                application/xml:
+                  description: Successful response payload
+                  type: !include schemas/wco/declaration/declarationInformationRetrievalStatusResponse.xsd
```

Comment:
How do they actually get the schemas? Do we need to provide a zip file like we do in customs-dec?

Reply:
build.sbt create the zip

Reply:
Is there a link to download them?